src/background.js

```diff
@@ -468,7 +466,6 @@ async function blockFacebookSubResources (requestDetails) {
     // See https://github.com/mozilla/contain-facebook/issues/23
     // Sometimes this add-on is installed but doesn't get a facebookCookieStoreId ?
     // eslint-disable-next-line no-console
-    console.log(error);
```

Comment:
We can use `console.error` here - our `eslint` rules allow it and it helps surface the error message.